use looper interface

agallou opened the pull request:
the runAgain behaviour were overload by checking the ask message
content : that's kind of an ugly hack.

agallou committed:
use looper interface

the runAgain behaviour were overload by checking the ask message
content : that's kind of an ugly hack.

agallou commented:
https://github.com/atoum/atoum/pull/567 has simplified the loop extraction https://github.com/atoum/atoum/pull/555. I've close the https://github.com/atoum/autoloop-extension/pull/2 which were using the https://github.com/atoum/atoum/pull/555 PR.

agallou commented:
when atoum's master will be tagged we will need to add this tag as minimum dependency for the autoloop extension.

jubianchi commented:
Seems good. I just tested the branch locally and it works fine!

One thing: [here](https://github.com/atoum/autoloop-extension/pull/4/files#diff-acdcc32f7f84a3a0128635b71b4a2d33R60) I would use `return true;` to make things clear.

:+1:

agallou committed:
move the minimum required version of atoum to 2.6.0

The improvements in https://github.com/atoum/autoloop-extension/pull/4
lies upon this PR on atoum : https://github.com/atoum/atoum/pull/567
which has been released on version 2.6.0.
